cmd/scollector/collectors/sql_windows.go

```diff
@@ -284,3 +301,113 @@ type Win32_PerfRawData_MSSQLSERVER_SQLServerDatabases struct {
 	TransactionsPersec            uint64
 	WriteTransactionsPersec       uint64
 }
+
+func c_mssql_replica_db() (opentsdb.MultiDataPoint, error) {
+	var dst []Win32_PerfRawData_MSSQLSERVER_SQLServerDatabaseReplica
+	var q = wmi.CreateQuery(&dst, "WHERE Name <> '_Total'")
```

Comment:
You can just use sqlAGDBQuery here, it's already been correctly set as that query.